src/masonite/providers/RouteProvider.py

```diff
@@ -59,7 +60,9 @@ def boot(self):
                     )
 
             else:
-                response.view("route not found", status=404)
+                raise RouteNotFoundException(
```

Comment:
Doesn't this need to check for debug mode?

Reply:
No that's the role of the exception handler. Depending of the exceptions and the debug mode it will handle this differntly